flang/test/Lower/array.f90

```diff
@@ -132,14 +132,14 @@ end subroutine range
 ! c0 array constructor
 ! CHECK: fir.global internal @_QQro.2x3xz4.{{.*}}(dense<{{\[}}[(1.000000e+00,1.500000e+00), (2.000000e+00,2.500000e+00)], [(3.000000e+00,3.500000e+00), (4.000000e+00,4.500000e+00)], [(5.000000e+00,5.500000e+00), (6.000000e+00,6.500000e+00)]]> : tensor<3x2xcomplex<f32>>) constant : !fir.array<2x3x!fir.complex<4>>
 
-! CHECK-LABEL rangeGlobal
+! CHECK-LABEL: rangeGlobal
```

Comment:
```suggestion
! CHECK-LABEL: rangeglobal
```

Reply:
Still failing CI at:
```
flang/test/Lower/array.f90:118:10: error: CHECK: expected string not found in input
flang/test/Lower/array.f90:137:10: error: CHECK: expected string not found in input 
```

Reply:
Ahh sorry for missing this. I think moving all of the fir.global checks to the end of the file fixes it: https://github.com/tblah/llvm-project/commit/b46391594d7246c6cd538b7a56238f209d3690ce

This happens because of the order in which we generate each MLIR operation. I suspect this test never worked properly.